web/server/codechecker_server/api/report_server.py

```diff
@@ -3424,10 +3424,24 @@ def removeRun(self, run_id, run_filter):
             # cascades and such. Deleting runs in separate transactions don't
             # exceed a potential statement timeout threshold in a DBMS.
             runs = []
+            run_no = 0
```

Comment:
I'm not sure `run_no` is a clear variable name. Maybe deleted_run_cnt or something else would be better.